<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>История рецептов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="histWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
